src/App.tsx

```diff
@@ -17,18 +98,34 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                onStatusChange={handleStatusChange}
+                onSearchChange={handleSearchChange}
+                onClearSearch={handleClearSearch}
+                searchQuery={searchQuery}
+              />
```

Comment:
Ensure that the `TodoFilter` component correctly accepts and types its props. This includes `onStatusChange`, `onSearchChange`, `onClearSearch`, and `searchQuery`.